xml/admin_monitoring_alerting.xml

```diff
@@ -1500,19 +1496,18 @@ ceph-volume lvm create --osd-id $N --data /path/to/device</screen>
         or by an external tool via the command:</para>
 <screen>&prompt.cephuser;ceph device set-life-expectancy <replaceable>DEVICE-ID</replaceable> <replaceable>FROM</replaceable> <replaceable>TO</replaceable></screen>
       <para>You can change the stored life expectancy manually, but that usually
-        does not remain&mdash;the tool that originally set it will
-        probably set it again, and changing the stored value does not affect
-        the actual health of the hardware device.</para>
+        does not remain&mdash;the tool that originally set it reset and changing
+        the stored value does not affect the actual health of the hardware device.</para>
       </listitem>
     </varlistentry>
     <varlistentry>
       <term><option>DEVICE_HEALTH_IN_USE</option></term>
       <listitem>
-        <para>One or more devices is expected to fail soon and has been marked
-          “out” of the cluster based on <literal>mgr/devicehealth/mark_out_threshold</literal>,
-          but it is still participating in one more PGs. This may be because it
-          was only recently marked “out” and data is still migrating, or because
-          data cannot be migrated off for some reason (for example, the cluster is
+        <para>One or more devices are expected to fail and has been marked
+          <option>out</option> of the cluster based on <literal>mgr/devicehealth/mark_out_threshold</literal>.
+          It is still participating in one more PGs. This may be because it
```

Comment:
"It" seems ambiguous here.

I think it's referring back to the "one or more devices" that are expected to fail soon. If I am correct then I suggest replacing the comma and rewording to:
"... based on mark_out_threshold, but the devices are still participating..."